react-frontend/src/components/CaseStudies/cardlist.js

```diff
@@ -14,13 +15,8 @@ const descriptions = [
   'In 2016, MSP received 130,000 enrolment and 147,680 Premium Assistance applications. Thirty percent of application forms were returned due to errors. To improve the process, the...',
 ];
 
-const imgLinks = [
-  'https://dalehartery.files.wordpress.com/2015/05/light-grey-background-pattern.jpg',
-  'https://dalehartery.files.wordpress.com/2015/05/light-grey-background-pattern.jpg',
-  'https://dalehartery.files.wordpress.com/2015/05/light-grey-background-pattern.jpg',
-];
-
 function CardList() {
```

Comment:
like to see this using the arrow functions. Lets be consistent with code style and modern js